audio/BlackHole/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+PortGroup           xcode_workaround 1.0
+
+github.setup        ExistentialAudio BlackHole 0.2.6 v
+homepage            https://github.com/ExistentialAudio/BlackHole
```

Comment:
The homepage line should be removed. It is already set to this by the `github.setup` command.